django/db/migrations/writer.py

```diff
@@ -175,7 +175,9 @@ def as_string(self):
 
         # Sort imports by the package / module to be imported (the part after
         # "from" in "from ... import ..." or after "import" in "import ...").
```

Comment:
I think the comment could be updated too, to reflect the new behavior.

Reply:
Thanks @David-Wobrock, great suggestion. I've pushed a commit that does that.